pkg/session/session.go

```diff
@@ -89,8 +92,7 @@ func GetOrCreate(
 	// Cache the session.
 	sessionCache[sessionKey] = session
 
-	// TODO(akutz) Reintroduce the logger.
-	//ctx.Logger.V(2).Info("cached vSphere client session", "server", server, "datacenter", datacenter)
+	logger.V(2).Info("cached vSphere client session", "server", server, "datacenter", datacenter)
```

Comment:
Would "created and cached" not ebe a bit more speaking to what it did?

Reply:
TBH yes that describes it more. I was just following akutz's initial thought process here. Either way it gives us the signal we are looking for.